Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -1391,13 +1391,28 @@ void RenderBox::clearOverridingLogicalWidthLength()
 
 FlowRelativeDirection RenderBox::physicalToFlowRelativeDirectionMapping(PhysicalDirection direction) const
 {
-    auto isHorizontalWritingMode = style().isHorizontalWritingMode();
-    auto isLeftToRightDirection = style().isLeftToRightDirection();
+    auto determineFormattingContextRootStyle = [&]() -> const RenderStyle& {
```

Comment:
auto determineFormattingContextRootStyle = [&] { ?

Reply:
Unfortunately the type deduction throws an error here and deduces just `RenderStyle`. Maybe there's another way to resolve that issue but I'm not super familiar with it.